sources/packages/backend/libs/services/src/sfas/sfas-application.service.ts

```diff
@@ -58,6 +58,21 @@ export class SFASApplicationService extends DataModelService<SFASApplication> {
       .getOne();
   }
 
+  /**
+   * Total BCSL amount that the student received from the legacy(SFAS) system.
+   * @param studentId student id.
+   * @returns total BCSL amount that the student received from the legacy(sfas) system.
+   */
+  async totalLegacyBCSLAmount(studentId: number): Promise<number> {
```

Comment:
As part of the upcoming ticket, we can also discuss the possibility of stopping doing the calculation every time and maybe have the SUM for the students consolidated in some other table on SIMS.